src/screens/status/status.vue

```diff
@@ -1,48 +1,89 @@
-          <td>{{ nodeStatus.top.time }}</td>
-          <td>{{ nodeStatus.version.revision | startAndEnd }}</td>
-          <td>{{ nodeStatus.version.genesisHash | startAndEnd }}</td>
-        </tr>
-      </table>
-      <h1>Detail</h1>
-      <h2>{{ node }} </h2>
-      <h3>version</h3>
-      <pre>{{ nodeStatus.version }}</pre>
-      <h3>top</h3>
-      <pre>{{ nodeStatus.top }}</pre>
+      <h2>Change Connected Network</h2>
+      <form @submit.prevent="changeNetwork">
+        <ae-input placeholder="Network to connect to: " v-model="newNetworkUrl" @keyup.enter="changeNetwork"/>
+        <ae-button
+          type="exciting"
+          size="medium"
+        >
+          connect
+        </ae-button>
+      </form>
+      <v-wait for="changing network">
+        <ae-loader slot="waiting" class="loader"/>
+        <h1>Status</h1>
+        <p>Explorer connected to: <strong>{{ node }}</strong></p>
+        <h2>Node and Peers</h2>
+        <table>
+          <tr>
+            <th>address</th>
+            <th>top.height</th>
+            <th>top.hash</th>
+            <th>top.time</th>
+            <th>version.revision</th>
+            <th>version.genesis_hash</th>
+          </tr>
+          <tr>
+            <td><strong>{{ node }}</strong></td>
+            <td>{{ nodeStatus.top.height }}</td>
+            <td>{{ nodeStatus.top.hash | startAndEnd }}</td>
+            <td>{{ nodeStatus.top.time }}</td>
+            <td>{{ nodeStatus.version.revision | startAndEnd }}</td>
+            <td>{{ nodeStatus.version.genesisHash | startAndEnd }}</td>
+          </tr>
+        </table>
+        <h1>Detail</h1>
+        <h2>{{ node }} </h2>
+        <h3>version</h3>
+        <pre>{{ nodeStatus.version }}</pre>
+        <h3>top</h3>
+        <pre>{{ nodeStatus.top }}</pre>
+      </v-wait>
     </div>
   </div>
 </template>
 <script>
 import { mapState } from 'vuex'
 import pollAction from '../../mixins/pollAction'
+import {
+  AeButton,
+  AeInput,
+  AeLoader
+} from '@aeternity/aepp-components'
 
 export default {
   data: function () {
-    return { node: process.env.VUE_APP_EPOCH_URL }
+    return { newNetworkUrl: '' }
   },
   mixins: [pollAction('getNodeStatus')],
   computed: mapState({
-    nodeStatus: '$nodeStatus'
-  })
+    nodeStatus: '$nodeStatus',
+    node: 'epochUrl'
+  }),
+  components: {
+    AeButton,
+    AeInput,
+    AeLoader
+  },
+  methods: {
+    async changeNetwork () {
+      this.$store.commit('changeNetworkUrl', this.newNetworkUrl)
+      this.newNetworkUrl = ''
+      this.$store.commit('blocks/resetState')
+      this.$store.commit('accounts/resetState')
+      this.$store.commit('transactions/resetState')
+
+      this.$store.dispatch('wait/start', 'changing network', { root: true })
+      await Promise.all([
+        this.$store.dispatch('blocks/height'),
+        this.$store.dispatch('getNodeStatus')
+      ])
+      this.$store.dispatch('wait/end', 'changing network', { root: true })
```

Comment:
I think for the local state of a component is more obvious to use data field and `v-if` than `vue-wait` package.